.github/workflows/helium/lib/createHeliumPkgFile.js

```diff
@@ -0,0 +1,41 @@
+import Promise from "bluebird";
+import stringify from 'json-stringify-pretty-compact';
+import {getMavenHandler} from "./getMavenArtifactInfo.js";
+import {fetchHelium} from "./fetchHeliumPkgInfo.js";
+import * as fs from "fs";
+import {getBasicMavenHandler} from "./getBasicMavenArtifactInfo.js";
+
+const pushPkgInfoToTmp = async function() {
+    // const response = await fetch('https://zeppelin.apache.org/helium.json')
+    // const data = await response.json();
+    // return data[0];
```

Comment:
```suggestion
```
If it's not necessary, could you please remove them?

Reply:
I miss that. I'll remove it and push again.